vfs/vfs.go

```diff
@@ -26,6 +25,26 @@ const (
 	FolderDocType = "io.cozy.folders"
 )
 
+// DocMetaAttributes is a struct containing modifiable fields from
+// file and directory documents.
+type DocMetaAttributes struct {
+	Name     string   `json:"name,omitempty"`
+	FolderID string   `json:"folderID,omitempty"`
+	Tags     []string `json:"tags,omitempty"`
+}
+
+// Context is used to convey the afero.Fs object along with the
+// CouchDb database prefix.
+type Context struct {
```

Comment:
[Future] This could get complicated fast between gin.Context, couchdb.Prefix (+ some auth later), vfs.Context, middlewares.Instance. We should at least have some out-of-code doc on it, but better a proper discussion and normalisation.